modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/configuration/FFAnalyticsReportsDetailsConfiguration.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+
+package com.liferay.analytics.reports.web.internal.configuration;
+
+import aQute.bnd.annotation.metatype.Meta;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+
+/**
+ * @author David Arques
+ */
+@ExtendedObjectClassDefinition(generateUI = false)
+@Meta.OCD(
+	id = "com.liferay.analytics.reports.web.internal.configuration.FFAnalyticsReportsDetailsConfiguration"
+)
+public interface FFAnalyticsReportsDetailsConfiguration {
+
+	@Meta.AD(deflt = "false", required = false)
+	public boolean referralEnabled();
```

Comment:
If the details part is important for you, you could add it here as referralDetailsEnabled.